charts/infra-apps/examples/thanos.yaml

```diff
@@ -1,7 +1,7 @@
 # Deploy Thanos
 #
-# This takes care of deploying Thanos' store gateway, query frontend, and receiver. 
-# Usually the prometheus-operator from kube-prometheus-stack will deploy 
+# This takes care of deploying Thanos' store gateway, query frontend, and receiver.
+# Usually the prometheus-operator from kube-prometheus-stack will deploy
```

Comment:
somehow pre-commit did some whitespace changes on files you didn't indend to touch...
```suggestion
# This takes care of deploying Thanos' store gateway, query frontend, and receiver. 
# Usually the prometheus-operator from kube-prometheus-stack will deploy 
```
I'm not sure why we've been seeing this on this repo lately, but am planning to  migrate to pre-commit.ci instead of the current setup which has exhibited these issues over the last couple of weeks.

(the files are indeed wrong but non-timed changes should not be in a timed pr)